src/PokemonTrainer.java

```diff
@@ -0,0 +1,7 @@
+import java.util.List;
+
+public class PokemonTrainer extends Player {
```

Comment:
Een PokemonTrainer voegt nu eigelijk niks toe aan Player., maar het is wel goed bedacht om er een abstracte klasse boven te zetten.